test/samples/typescript-block-filenames/.eslintrc.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
 		},
 	],
 	settings: {
-		'svelte3/typescript': require('typescript'),
+		'svelte3/typescript': () => require('typescript'),
```

Comment:
Could you revert the changes to the existing tests and add a new test which tests the lazy loading? Like `typescript-lazy-load`

Reply:
Yes nice idea 👍

Reply:
done 😉